pkg/provider/bitbucketcloud/bitbucket.go

```diff
@@ -31,6 +31,10 @@ type Provider struct {
 	provenance    string
 }
 
+func (v *Provider) CreateComment(_ context.Context, _ *info.Event, _, _ string) error {
```

Comment:
Should this be documented and/or log as unsupported? 
E.g 
```suggestion
func (v *Provider) CreateComment(_ context.Context, _ *info.Event, _, _ string) error {
    log.Warn("Comment Creation is not supported for Bitbucket Cloud")
```

Reply:
Also, is support parity here just a low priority or is it not possible? If it's possible but just not a priority, should there maybe be a TODO?

Reply:
it's just low priority until more customer are vouching to use bitbucketcloud, we don't have that many (or any) on it so it's a best effort basis, document the features parity is important